ui/StatusQ/src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -1,195 +1,129 @@
-                return communityListView.model.count * communityNavBarButtonHeight +
-                        numOfSpacingsForCommunities * navBarButtonSpacing +
-                        separatorBottom.height
-            }
-
-            StatusAppNavBarFilterModel {
-                id: navBarCommunityModel
-
-                filterAcceptsItem: filterCommunityItem
-
-                model: statusAppNavBar.sectionModel
-
-                delegate: communityNavBarButton
-
-                onAboutToUpdateFilteredModel: {
-                    statusAppNavBar.aboutToUpdateFilteredCommunityModel()
-                }
-            }
-
-            Item {
-                id: separatorTop
-                width: parent.width
-                height: navBarButtonSpacing
-                anchors.top: parent.top
-                visible: parent.necessaryHightForCommunities > parent.maxHightForCommunities
-
-                Rectangle {
-                    height: 1
-                    width: 30
-                    color: Theme.palette.directColor7
-                    anchors.top: parent.top
-                    anchors.horizontalCenter: parent.horizontalCenter
-                }
-            }
-
-            ListView {
-                id: communityListView
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.top: separatorTop.visible? separatorTop.bottom : parent.top
-                anchors.bottom: separatorBottom.top
-                clip: true
-
-                boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
-                spacing: navBarButtonSpacing
-
-                model: navBarCommunityModel
-            }
-
-            Item {
-                id: separatorBottom
-                width: parent.width
-                height: navBarButtonSpacing
-                anchors.bottom: parent.bottom
-
-                Rectangle {
-                    height: 1
-                    width: 30
-                    color: Theme.palette.directColor7
-                    anchors.bottom: parent.bottom
-                    anchors.horizontalCenter: parent.horizontalCenter
-                }
-            }
+            Layout.fillWidth: true
+            Layout.preferredHeight: d.implicitListViewHeight(this)
+
+            objectName: "statusMainNavBarListView"
+
+            clip: true
+            spacing: d.spacing
+            boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
         }
-    }
 
-    ListView {
-        id: navBarListView
-        objectName: "statusMainNavBarListView"
-        anchors.left: parent.left
-        anchors.right: parent.right
-        anchors.top: parent.top
-        anchors.topMargin: 48
-        anchors.bottom: navBarProfileButtonSlot.top
-        anchors.bottomMargin: navBarButtonSpacing
+        Item {
```

Comment:
What does this Item do?

Reply:
This is a common practice to separate items in layouts, you can think of it as an invisible spacer.
When you resize the layout it behaves as below:

![image](https://user-images.githubusercontent.com/33099791/204353388-df9576e2-f488-42ba-981d-f65f22fad826.png)